src/xcode/ENA/ENA/Source/Extensions/StringInterpolation+Private.swift

```diff
@@ -0,0 +1,19 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+	
+extension String.StringInterpolation {
+	/// Use this method when logging sensitive data. This ensures, that the logged information is censored in release builds, but not in debug build, which is needed for us devs and the testers.
+	/// - Parameters:
+	///   - private: The sensitive data to be replaced in release builds with '🙈🙉🙊'.
+	///   - public: Additional explination text what should be logged as data but is censored.
+	mutating func appendInterpolation<T>(private value: T, public text: String = "") {
+		#if DEBUG
+			appendLiteral(String(describing: value))
+		#else
+			text.isEmpty ? appendLiteral("🙈🙉🙊") : appendLiteral("🙈🙉🙊. (Censoring cause: " + text + ")")
```

Comment:
I would keep it simple and just use the alternative `text` as is.

Reply:
See comment above